app/models/copilot/content.rb

```diff
@@ -0,0 +1,11 @@
+module Copilot
+  class Content < ActiveRecord::Base
+    validates :slug, presence: true, uniqueness: true
+
+    def self.fetch(slug, value='')
+      content = self.find_or_create_by(slug: slug)
+      content.update(value: value) unless content.value.present?
```

Comment:
If there is no value in the database, than populate it from the html/browser.  If you don't do this, then the value is not preserved since it always gets overridden by the default value in the html.  The fetch and the update are 2 different use cases, this is set up so that the db gets populated when the views are rendered.